### PR TITLE
Tukan / Cicada - small color fixes

### PR DESCRIPTION
**Pull request by opnsenseuser**
small improvements for .text-danger, .btn-danger and .fw_block classes

### CODE REVIEW

File: misc/theme-cicada/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-cicada
-PLUGIN_VERSION=		1.13
+PLUGIN_VERSION=		1.13_1
```
Comment (fichtner) [inline misc/theme-cicada/Makefile]:
```suggestion
PLUGIN_VERSION=		1.14
```

File: misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/main.scss [hunk 2]
```diff
@@ -9717,4 +9717,4 @@ h3 {
 #chart, #chart_intf_in, #chart_intf_out, #chart_top_ports, #chart_top_sources, #traffic_graph_widget_chart_in, #traffic_graph_widget_chart_out {
   background-color: #292929;
   border: 1px solid #515151;
-}
+}
```
Comment (fichtner) [inline misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/main.scss]:
```suggestion
}
```

File: misc/theme-tukan/Makefile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-tukan
-PLUGIN_VERSION=		1.12
+PLUGIN_VERSION=		1.12_1
```
Comment (fichtner) [inline misc/theme-tukan/Makefile]:
```suggestion
PLUGIN_VERSION=		1.13
```

**Comment by opnsenseuser** [comment]
@fichtner no change to get this in the new release today? Regards rene

**Comment by fichtner** [comment]
19.1 was built on Monday morning to allow for a longer testing period. But first things first, you need to apply the suggestions...

**Comment by opnsenseuser** [comment]
Done

**Comment by fichtner** [comment]
Merged, thanks! 19.1.1 will follow rather quickly this time so that this can be included.

**Comment by fichtner** [comment]
and thank you for the continuing submissions ❤️

**Pull request merged**